### ISSUE TITLE
improve usability and documentation of kube-score --ignore-test

### ISSUE THREAD

**Issue by cruschke** [zegl/kube-score#70]
**Describe the bug**
To understand what are the valid values for `kube-score --ignore-test` you have either to read the source code or guess what might be a correct one. 

**To Reproduce**
Try to suppress some failed checks without knowing what are the options

**Expected behavior**
Having a list of valid values. 

Ideally you could also give each check a UID as in [hadolint](https://github.com/hadolint/hadolint#rules) or [shellcheck](https://github.com/koalaman/shellcheck/wiki/Checks) and suppress them with `kube-score --ignore-test KS0017, KS0033`, a `.kube-score.yaml`

```yaml
ignored:
  - KS0017
  - KS0033
```
or as ENV like KUBESCORE_OPTS="--ignore-test KS0017, KS0033".

(Concepts shamelessly stolen at `hadolint` and `shellcheck` :smiley:)

**Version**
kube-score version: 0.4.0

**Additional context**
These are the ones that I found (as reminder to myself)
* container-image-pull-policy
* container-image-tag
* container-resources
* container-security-context
* cronjob-has-deadline
* deployment-has-poddisruptionbudget
* ingress-targets-service
* networkpolicy-targets-pod
* pod-networkpolicy
* pod-probes
* service-targets-pod
* service-type
* stable-version
* statefulset-has-poddisruptionbudget

**Comment by zegl** [comment]
Hey @cruschke! Thanks for creating this issue, I agree with you that there is some documentation missing here. I'll create a document with a list/table of all available tests similar to how hadolint is doing it.

I'll leave improvements to configuring the ignores for another time, as it's a bit unclear what the best way to do it is.

**Comment by cruschke** [comment]
And @zegl thanks for writing this great piece of software :+1:

**Comment by zegl** [comment]
I have now added a `kube-score list` sub-command, that prints a list and a short description of all checks. The sub-command feature is a bit experimental, so I'm going to keep this issue open until it's stable and a new release has been made with it enabled.

From that sub-command I've generated the file [README_CHECKS.md](https://github.com/zegl/kube-score/blob/master/README_CHECKS.md), which is now linked from the main README.
